cri/v1alpha2/cri.go

```diff
@@ -774,6 +774,7 @@ func (c *CriManager) ContainerStatus(ctx context.Context, r *runtime.ContainerSt
 		Volumes:     parseVolumesFromPouch(container.Config.Volumes),
 		Resources:   parseResourcesFromPouch(resources, diskQuota),
 		QuotaId:     container.Config.QuotaID,
+		Envs:        parseEnvsFromPouch(container.Config.Env),
```

Comment:
I do not think we should add `Pouch` in the function naming. Please change a proper one. @Starnop

Reply:
@Starnop Maybe we should rename the functions like `parseEnvsFromPouch`, `parseResourcesFromPouch` in another PR, they all do the same thing :)

@allencloud WDYT?

Reply:
:+1: I will do that in the next pull request.

Reply:
I still insist that we could finish in the pull request. How about `convertEnvsForCRI` or `convertContainerEnvForCRI`?